samples/model-builder/model_registry/upload_new_aliased_model_version_sample.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_model_registry_upload_new_aliased_model_version_sample]
+
+from typing import List
+
+from google.cloud import aiplatform
+
+
+def upload_new_aliased_model_version_sample(
+    model_id: str,
+    artifact_uri: str,
+    serving_container_image: str,
+    is_default_version: bool,
+    version_aliases: List[str],
+    version_description: str,
+    project: str,
+    location: str,
+):
+    """
+    Uploads a new aliased version of a model with ID 'model_id'.
+    Args:
+        model_id: The ID of the model to upload a new version. Parent resource name of the model is also accepted.
+        artifact_uri: The URI of the model artifact to upload.
+        serving_container_image: The name of the serving container image to use.
+        is_default_version: Whether this version is the default version of the model.
```

Comment:
s/is/should be?

Reply:
I only use the minimum amount of parameters to upload the model

Reply:
I meant shall we change the document to 

is_default_version: Whether this version should be the default version of the model.